packages/samples/headless-commerce-ssr-remix/client/external-context-api.ts

```diff
@@ -0,0 +1,44 @@
+export type ExternalContextInformation = {
```

Comment:
The idea here is to show that for the purpose of formatting prices and such on interfaces that are not mainly powered by Coveo (e.g., PDP, cart), the source of truth for context info (in this case, the locale) should come from an external service.